server/src/headers/Database/Job.h

```diff
@@ -149,6 +149,7 @@ class CJob {
         std::vector<Config::Ptr<CDictionary>> m_dictionaries;  /**< Vector of non-exhausted dictionaries */
     private:
         std::string m_hashes;           /**< job hash(es) denoted by newlines */
+        std::string m_all_hashes;           /**< job hash(es) denoted by newlines */
```

Comment:
This is strange. Comment is same as m_hashes. Purpose?

Needs to be explained!